protocol/x/prices/keeper/update_price.go

```diff
@@ -134,6 +121,13 @@ func (k Keeper) GetValidMarketPriceUpdates(
 		}
 	}
 
+	if len(nonExistentMarkets) > 0 {
+		ctx.Logger().Warn(
```

Comment:
Prices are managed by Slinky sidecar and this log rarely has material signal, so use `Warn` instead